satellite/metainfo/batch.go

```diff
@@ -25,6 +25,9 @@ func (endpoint *Endpoint) Batch(ctx context.Context, req *pb.BatchRequest) (resp
 		switch singleRequest := request.Request.(type) {
 		// BUCKET
 		case *pb.BatchRequestItem_BucketCreate:
+			if singleRequest.BucketCreate.Header == nil {
```

Comment:
Unfortunate that the oneof types don't implement some general getter/setter that could be used to simplify this code. e.g. `if req, ok := request.Request.(interface{ GetHeader()*pb.RequestHeader; SetHeader(*pb.RequestHeader) }); ok {` would be neat.

Reply:
yeah seriously